### ISSUE TITLE
Chris keeps popping up to say hi

### ISSUE THREAD

**Issue by libeclipse** [keybase/client#3301]
I opened chris's public folder to test the app the first time I ran it, and now strangely the little window that accompanies the viewing of the folder keeps randomly popping up. (Weirdly when I have visual code open but that's probably a coincidence.) I tried to delete chris's folder from my local thing but it didn't work.

I mean the dude's nice and all but it's getting a little creepy. :P

**Comment by cjb** [comment]
> (Weirdly when I have visual code open but that's probably a coincidence.)

I wonder if visual code is crawling /keybase (and therefore /keybase/public/chris) for code..

**Comment by libeclipse** [comment]
That could maybe explain it. I'll see if it carries on happening when I open VS.

Is there any current way of removing or "forgetting" that I opened chris's public folder in the first place?

**Comment by cjb** [comment]
yep, `rmdir /keybase/public/chris` should do it

**Comment by libeclipse** [comment]
It didn't work the last 5 times I tried it, but it just worked now. Thanks.

**Comment by libeclipse** [comment]
@cjb Also probably related: I switched to using Atom, and just now when I opened the editor, I was prompted for my password again by keybase, even though I had already previously mounted it on boot. Thoughts on why this might have happened?

**Comment by libeclipse** [comment]
@cjb Update: Chris is still there. He reappeared in /keybase/public and popped up again but this time when I was searching through /home/eclipse for a project folder in Atom. (So when the dialog for opening a file was open)

**Comment by strib** [comment]
Hi @libeclipse -- probably something on your computer ended up indexing and remembering that path, and periodically looks for it (which causes it to reappear).  There's not a way around that right now without finding whatever program it is and, if it's possible, manually removing that path from the index.

However, you should be able to stop the chris popup by tracking `chris`, either by clicking on the `track` button in the popup, or running `keybase track chris`.

**Comment by libeclipse** [comment]
Issue also happened when I opened nemo as root.

@strib Alright thank you. I'll open a separate issue for not being able to delete Chris's folder.

**Comment by strib** [comment]
@libeclipse it's not really a bug.  It sounds like `rmdir` is successfully removing it, but something on your device (like an indexing program) remembers the path and accesses it again.  KBFS can't tell the difference between you intentionally doing that, and some program running as you doing it, so it re-creates the folder when that happens.

Perhaps eventually we'll have a way for you to ignore a folder for all time, and fail all future accesses to it, but I'm not sure it's a good idea to have something so irreversible.

**Comment by libeclipse** [comment]
@strib Ah. I understand. Well it happened mostly when I open a text editor or nemo or something. Not sure where to start looking though. Shall I close the related issue?

**Comment by strib** [comment]
Up to you -- if you feel like it's an important feature we should have, you can leave it open and see if other people agree, which might convince us to prioritize something like that.  Sorry for the frustrating experience.

**Comment by libeclipse** [comment]
It's alright, I know all too well about how horrible it can be when one of
your users has an obscure problem that no one else can recreate. :P

I agree that having something so irreversible is notagoodthing, but I need
help figuring out how to track down what's causing this. Maybe in the
future it can be detected and ignored or similar.

**Comment by juanminm** [comment]
I would say this is the same issue, but being a different OS with different KBFS implementation I could be wrong. It does as @libeclipse explained: Remove the folder but later it reappears again with a pop-up.

OS: Windows 10, 1151, 64-bit

**Comment by libeclipse** [comment]
That does sound like the same issue. If it is, then there's something on
both Arch Linux and Windows 10 that's caching the folder and crawling the
keybase directory for it later, even after it's deleted. I haven't seen
anything like it before.

**Comment by MannyC** [comment]
This was happening to me on every boot, and I appear to have fixed the issue.

As mentioned above, it was caused by a process remembering the path and trying to access it in the background.

To find out which process I used `cat ~/.cache/keybase/keybase.kbfs.log | grep 'Pid.*chris'`

For me it turned out to be "zeitgeist-datahub", something I had no idea was running and certainly don't want.    After disabling it via the Activity Log Manager, the popup hasn't appeared since.

**Comment by wybiral** [comment]
Yep, something on my computer (Ubuntu 16.04) is doing the same thing. I'm thinking it's either Gedit or VSCode. In this case it isn't that annoying but it could get bad if you browsed enough people's public folders for it to happen regularly.
